apps/frontend/src/app/poll/choose-events/choose-events.component.ts

```diff
@@ -222,11 +212,9 @@ export class ChooseEventsComponent implements OnInit {
   }
 
   private clearSelection(){
-    this.name = '';
-    this.checks = new Array(this.checks.length).fill(CheckboxState.FALSE);
-  }
-
-  private filterEvents(checks: CheckboxState[], state: CheckboxState) {
-    return this.pollEvents.filter((_, i) => checks[i] === state).map(e => e._id);
+    this.newParticipant.name = '';
```

Comment:
This works, but I'm getting `Attempt to assign to const or readonly variable`.
You should refactor the method as i suggested you to.

Reply:
This is an IDE problem. It compiles fine.